journal.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from pyramid.config import Configurator
+from pyramid.view import view_config
+from waitress import serve
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+
+Base = declarative_base()
+
+# add this import at the top:
+import datetime
+
+# and define your class below the Base
+class Entry(Base):
+    __tablename__ = 'entries'
+    id = sa.Column(sa.Integer, primary_key=True, autoincrement=True)
+    title = sa.Column(sa.Unicode(127), nullable=False)
+    text = sa.Column(sa.UnicodeText, nullable=False)
+    date = sa.Column(
+        sa.DateTime, nullable=False, default=datetime.datetime.utcnow
+    )
+
+DATABASE_URL = os.environ.get(
+    'DATABASE_URL',
+    'postgresql://johnson:password@localhost:5432/learning-journal'
+)
+
+def init_db():
```

Comment:
your linter should be showing you a warning on this line of code.  There's only one space between it and the previous block.  Module-level blocks should be separated by two blank lines.